src/models/connectionStore.ts

```diff
@@ -391,11 +391,14 @@ export class ConnectionStore {
     }
 
     // Load connections from user preferences
-    public loadAllConnections(): IConnectionCredentialsQuickPickItem[] {
+    public loadAllConnections(addRecentConnections: boolean = true): IConnectionCredentialsQuickPickItem[] {
```

Comment:
it seems like all invocations of this method are calling with false.  Maybe we should make the default false?

Reply:
Only in the case of OE features. For queries, people still might use the MRU.